packages/astro/src/types/public/config.ts

```diff
@@ -1283,7 +1284,7 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 		 * }
 		 * ```
 		 */
-		syntaxHighlight?: 'shiki' | 'prism' | false;
+		syntaxHighlight?: SyntaxHighlightConfig | SyntaxHighlightConfig['type'] | false;
```

Comment:
I completely forgot about this one.

The JsDoc must be updated to reflect the new changes

Reply:
Updated JSDoc.